apps/web/app/(app)/gardens/[chain]/[garden]/page.tsx

```diff
@@ -22,93 +30,95 @@ export default async function Garden({
+    urqlClient,
+    chain,
+    getGardenDocument,
+    { addr: garden },
+  );
 
   let communities = result?.tokenGarden?.communities || [];
 
   communities = communities.filter((com) => com.isValid);
 
   const tokenGarden = result?.tokenGarden as TokenGarden;
 
-  const fetchAndUpdateCommunities = async (communities: any) => {
-    const promises = communities.map(async (com: any) => {
-      if (com?.covenantIpfsHash) {
-        const ipfsHash = com.covenantIpfsHash;
-        try {
-          const response = await fetch("https://ipfs.io/ipfs/" + ipfsHash);
-          const json = await response.json();
-          // Return a new object with the updated covenantIpfsHash
-          return { ...com, covenantData: json };
-        } catch (error) {
-          console.log(error);
-          // Return the original community object in case of an error
-          return com;
-        }
-      }
-      // Return the original community object if there's no covenantIpfsHash
-      return com;
-    });
+  const gardenTotalMembers = () => {
+    const uniqueMembers = new Set();
 
-    // Wait for all promises to resolve and update the communities array
-    const updatedCommunities = await Promise.all(promises);
-    return updatedCommunities;
-  };
+    communities.forEach((community) =>
+      community.members?.forEach((member) =>
+        uniqueMembers.add(member?.memberAddress),
+      ),
+    );
 
-  // Use the function
-  await fetchAndUpdateCommunities(communities)
-    .then((updatedCommunities) => {
-      communities = updatedCommunities; // Here you'll have the communities array with updated covenantIpfsHashes
-    })
-    .catch((error) => {
-      console.error("An error occurred:", error);
-    });
+    return uniqueMembers.size;
+  };
 
   return (
-    <div className="relative mx-auto max-w-6xl space-y-10 rounded-xl border-2 border-black bg-base-100 bg-surface p-8">
-      <header className="relative flex min-h-[500px] flex-col items-center justify-between gap-6 px-3">
-        <div className="flex h-full min-h-96 flex-col items-center justify-between p-1">
-          <h3 className="text-center font-press">
-            {tokenGarden?.symbol} Token Ecosystem
-          </h3>
-          <p className="max-w-lg text-center font-semibold leading-7">
-            Discover communities in the
-            <span className="text-primary"> {tokenGarden?.name} Garden</span>,
-            where you connect with people and support proposals bounded by a
-            shared
-            <span className="text-primary"> covenant.</span>
-          </p>
-          <FormLink
-            label="Create Community"
-            href={`/gardens/${chain}/${garden}/create-community`}
-          />
+    <div className="page-layout">
+      <header className="section-layout flex gap-10 p-10">
+        <div className="flex h-[280px] w-[300px] items-center justify-center rounded-2xl bg-slate-200">
+          Token Image
```

Comment:
Why not putting the figma image as placeholder ? 
![image](https://github.com/1Hive/gardens-v2/assets/13890618/4214ea1b-9481-42c9-9ea3-a9e5e7e021c2)

Reply:
got it